Source/Evolve/WarpXEvolve.cpp

```diff
@@ -284,49 +222,7 @@ WarpX::Evolve (int numsteps)
             }
         }
 
-        mypc->ContinuousFluxInjection(cur_time, dt[0]);
```

Comment:
I'm not sure that it is helpful to put this single line calls into a separate routine. For me, this would make it more difficult to follow what the evolve step is doing since these would be hidden.

Reply:
I think you might have misread, `InjectRedistributeScapeParticles` includes:
- injection,
- particle BC
- scraping
- redistribute

for particles. From the doc string: `Inject, communicate, scape and sort particles`

Reply:
I understand what this is doing - my comment is just that having the calls here makes it easier to following what is happening without having to jump to some other routine to see it.

All of these actions are handling particles at the boundaries. Perhaps call the routine `HandleParticlesAtBoundaries` (or something less clunky).

Further, almost all of these are using `mypc` so this new function could be put into that class. And also put `m_particle_boundary_buffer` into that class as well since it is particle related.

Reply:
Rename sounds great - applied.

Moving to the other class is a good idea, let's do in a follow-up to keep this simple here.